py_script/process_ings.py

```diff
@@ -1,15 +1,25 @@
+"""
+Used to process the raw data about ingredients pulled from the API.
 
+Usage: 
+- python process_ings.py [infile] [outfile] 
+OR
+- python process_ings.py [infile and outfile]
+"""
 
 import json
+import sys
+import os
+import base64
 
-with open("../ingreds.json", "r") as infile:
-    ing_data = json.loads(infile.read())
-ings = ing_data
-#this data does not have request :)
+infile, outfile = sys.argv[1], sys.argv[2] if len(sys.argv) > 2 else sys.argv[1]
```

Comment:
for refactor purposes, prolly use python's builtin argparse module -- might as well